doc/usersguide/source/model.rst

```diff
@@ -239,14 +243,14 @@ functions but not stored in the output file.
                 - age:      int
                 - agegroup: {type: int, initialdata: False}
                 - temporary: {type: int, output: False}
-
+                - alive: {type: int, initialdata: True}

```

Comment:
Sorry I will fix that.

On Thu, Nov 12, 2015 at 10:23 PM Gaëtan de Menten <notifications@github.com>
wrote:

> In doc/usersguide/source/model.rst
> <https://github.com/liam2/liam2/pull/190#discussion_r44716573>:
>
> > @@ -239,14 +243,14 @@ functions but not stored in the output file.
> >                  - age:      int
> >                  - agegroup: {type: int, initialdata: False}
> >                  - temporary: {type: int, output: False}
> > -
> > +                - alive: {type: int, initialdata: True}
>
> I guess you meant {type: int, default: True}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/liam2/liam2/pull/190/files#r44716573>.
>